[2022-09-13 20:16:40,662] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.notify scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2022-09-13 20:16:40,676] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_dag.notify scheduled__2022-09-10T00:00:00+00:00 [queued]>
[2022-09-13 20:16:40,677] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:16:40,678] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-13 20:16:40,679] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-13 20:16:40,700] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): notify> on 2022-09-10 00:00:00+00:00
[2022-09-13 20:16:40,708] {standard_task_runner.py:52} INFO - Started process 215 to run task
[2022-09-13 20:16:40,714] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'notify', 'scheduled__2022-09-10T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/pre_parser.py', '--cfg-path', '/tmp/tmpzc_cjv29', '--error-file', '/tmp/tmpfo5slauf']
[2022-09-13 20:16:40,716] {standard_task_runner.py:80} INFO - Job 17: Subtask notify
[2022-09-13 20:16:40,799] {task_command.py:369} INFO - Running <TaskInstance: test_dag.notify scheduled__2022-09-10T00:00:00+00:00 [running]> on host 5d92941617fc
[2022-09-13 20:16:40,885] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=notify
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-10T00:00:00+00:00
[2022-09-13 20:16:40,890] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-13 20:16:40,893] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hello!"']
[2022-09-13 20:16:40,926] {subprocess.py:85} INFO - Output:
[2022-09-13 20:16:40,929] {subprocess.py:92} INFO - Hello!
[2022-09-13 20:16:40,931] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-13 20:16:40,991] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=notify, execution_date=20220910T000000, start_date=20220913T201640, end_date=20220913T201640
[2022-09-13 20:16:41,047] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-13 20:16:41,196] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
